AGOL_updater_LTCF_Data.py

```diff
@@ -68,7 +68,7 @@ def locate(self, street, zone, **kwargs):
 
 # Function to send fields to geocoder, get x/y values back
 def geocode(row):
-    self = 'AGRC-XXXXXXXXXXXXX'     # insert correct API token here (home)
+    self = 'AGRC-08077BBD392574'     # insert correct API token here (home)
```

Comment:
Let's remove the API key...we don't want to publish it on GitHub.  Just re-insert the XXXXXs on your branch and resubmit the pull request.  Then insert the actual API key in the code you have locally on your machine.